### ISSUE TITLE
Black Screen on Android

### ISSUE THREAD

**Issue by dschuerrer** [google-ar/arcore-unity-extensions#181]
Hi, I am building an AR application based on the sample scene "GeospatialArf5", using Unity 2022.3.9f1. 
The versions of the installed packages are attached. 
the Android build works only after adding "Custom Main Gradle Template", however, when installed on Google Pixel 4a the app starts up, but the screen stays black. I have checked the URP settings, everything should be fine.
Thank you. 
<img width="329" alt="Bildschirmfoto 2024-03-05 um 15 57 14" src="https://github.com/google-ar/arcore-unity-extensions/assets/131254668/90645f2d-2505-4878-b961-ebe17928f7f2">
<img width="332" alt="Bildschirmfoto 2024-03-05 um 15 57 31" src="https://github.com/google-ar/arcore-unity-extensions/assets/131254668/6eb512ce-c99f-4e7e-8cb4-52096816891c">
<img width="427" alt="Bildschirmfoto 2024-03-05 um 16 02 14" src="https://github.com/google-ar/arcore-unity-extensions/assets/131254668/d0077221-c5f4-43a1-a043-9aca2640430a">

**Comment by devbridie** [comment]
As you mentioned URP, could you ensure that the [AR Camera Background](https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@4.2/manual/ar-camera-background-with-scriptable-render-pipeline.html) is properly configured? Otherwise you might want to check the logcat for any errors.

**Comment by dschuerrer** [comment]
It worked now, I had deactivated the canvas in the sample scene, which seemed to have been the problem. I have an AR camera view now, but now the application is stuck on "Localizing your device to set anchor" after a warning that VPS is not available and GPS will be used. Is that a new issue or can you comment in this thread? Thank you!

**Comment by dschuerrer** [comment]
mmh, ok thank you. I am aware of that and tried different locations of outside places that are actually included in street view, so thats strange.

**Comment by devbridie** [comment]
It's difficult to say without knowing the places you're talking about in particular, but one caveat is that [user generated street view](https://www.google.com/streetview/contribute/) is not compatible, so it could be that!
